lxc/storage_volume.go

```diff
@@ -2181,7 +2181,7 @@ func (c *cmdStorageVolumeExport) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Get name of backup
-	backupName := strings.TrimPrefix(op.Get().Resources["backups"][0],
+	backupName := strings.TrimPrefix(op.Get().Resources["backups"][0].String(),
 		"/1.0/backups/")
```

Comment:
If you find examples of `1.0` being hardcoded, please can you change them to use the constant. Thanks